packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -209,13 +222,19 @@ export class TokenRatesController extends BaseController<
    */
   constructor(
     {
+      interval = 3 * 60 * 1000,
+      threshold = 6 * 60 * 60 * 1000,
+      getNetworkClientById,
       chainId: initialChainId,
       ticker: initialTicker,
       selectedAddress: initialSelectedAddress,
       onPreferencesStateChange,
       onTokensStateChange,
       onNetworkStateChange,
     }: {
+      interval?: number;
+      threshold?: number;
+      getNetworkClientById: NetworkController['getNetworkClientById'];
```

Comment:
nit: see https://github.com/MetaMask/core/pull/2015/files#r1393439534

```suggestion
      getNetworkClientById: NetworkControllerGetNetworkClientByIdAction['handler'];
```